include/sys/zfs_znode.h

```diff
@@ -280,9 +280,13 @@ do {								\
 #define	ZFS_ENTER(zfsvfs)	ZFS_ENTER_ERROR(zfsvfs, EIO)
 #define	ZPL_ENTER(zfsvfs)	ZFS_ENTER_ERROR(zfsvfs, -EIO)
 
-/* Must be called before exiting the operation. */
+/*
+ * Must be called before exiting the vop
+ *
+ */
 #define	ZFS_EXIT(zfsvfs)					\
 do {								\
+	zfs_exit_fs(zfsvfs);					\
```

Comment:
nit: Sorry to make you rebase this again, but as part of the ongoing refactoring we've moved these macros in to a platform specific `include/os/linux/zfs/sys/zfs_znode_impl.h` header.  One nice upshot of this is it does provide us more flexibility in adding Linux specific logic to the macro.

Reply:
Done :)